ansible_rulebook/builtin.py

```diff
@@ -932,3 +933,11 @@ def _embellish_internal_event(event: Dict, method_name: str) -> Dict:
     return insert_meta(
         event, **dict(source_name=method_name, source_type="internal")
     )
+
+
+def _controller_job_url(data: dict) -> str:
+    if "id" in data:
+        return (
+            f"{job_template_runner.host}/#/jobs/playbook/{data['id']}/details"
```

Comment:
Why `playbook` is in the path?

Reply:
@bzwei that's the way the Controller UI shows it. The api path is different from the ui path.

Reply:
@bzwei it works both ways I removed the playbook which causes the tower to issue a redirect. Since we might be running workflows in the future its better to remove the playbook from the path.